app/code/Magento/NewsletterApi/Exception/AlreadySubscribedException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
```

Comment:
I'm agreeing a little bit with both of you.

On the one hand I would always prefer a specialized exception over a generic one.

On the other hand, in the service contract it might be better to be a little more generic while in the implementation itself the integrator can then introduce more specialized exceptions, that are extended from the more generic exception